bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -710,6 +697,58 @@ impl Client {
     pub fn encryption(&self) -> Arc<Encryption> {
         Arc::new(self.inner.encryption().into())
     }
+
+    // Ignored users
```

Comment:
Is this the right way to separate code blocks in Rust? 🤔

Reply:
I don't know of a convention here. You could put things into a separate `impl Client` block if you want to fully separate things, but I don't think that's necessary.